File: Examples/VisualBasic/Articles/Implement1904DateSystem.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles
    Public Class Implement1904DateSystem
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            ' Initialize a new Workbook
            ' Open an excel file
            Dim workbook As New Workbook(dataDir & "book1.xlsx")

            ' Implement 1904 date system
            workbook.Settings.Date1904 = True

            ' Save the excel file
            workbook.Save(dataDir & "output.xlsx")
            ' ExEnd:1



        End Sub
    End Class
End Namespace

File: Examples/CSharp/Articles/Implement1904DateSystem.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class Implement1904DateSystem
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Open an excel file
            Workbook workbook = new Workbook(sourceDir + "sampleImplement1904DateSystem.xlsx");

            // Implement 1904 date system
            workbook.Settings.Date1904 = true;

            // Save the excel file
            workbook.Save(outputDir + "outputImplement1904DateSystem.xlsx");

            Console.WriteLine("Implement1904DateSystem executed successfully.");
        }
    }
}
